src/containers/Review/ReviewWrapper.tsx

```diff
@@ -155,7 +156,9 @@ const ReviewHomeHeader: React.FC<ReviewHomeProps> = ({
   const [prevQueryString] = useState(location?.state?.prevQuery)
 
   if (!tab) {
-    updateQuery({ tab: tabIdentifiers[0] })
+    !isMobile
+      ? updateQuery({ tab: tabIdentifiers[0] })
+      : updateQuery({ tab: tabIdentifiers[tabIdentifiers.length - 1] })
```

Comment:
This makes 'overview' the default tab again in mobile as we are reversing the tab panes in mobile.